### DISCUSSION TITLE
tsh login - Object not found, Users can login using web UI but not tsh client

### DISCUSSION THREAD

**Discussion by Reab0** [gravitational/teleport#7478] [Q&A] [answered]
On one of the linux devices, after `tsh login`, users enter the password and otp code then they get this message: 

STEPS:
> Enter password for Teleport user username:
> Enter your OTP token:
> ******
> WARNING: You are using insecure connection to SSH proxy 
> **ERROR: object not found**

When I check the logs they have been authenticated successfully:
Logs: 

> Local Login | Local user [Username] successfully logged in

Note that this happens only using certain devices not all. and it worked previously on same devices but not anymore

We tried tsh logout and then login but same error.
Teleport version: Teleport v6.2.0 git:v6.2.0-0-g95b04b35a go1.15.5 

Thanks

**Comment by Reab0** [comment, marked as answer]
**Deleted** ~/.tsh directory content and it worked

**Reply by marieclairembanga** [reply]
Hey I've the same problem plz where is that .tsh directory exactly?

**Reply by webvictim** [reply]
`~` is your home directory.

Check `/Users/username` on MacOS, `/home/username` on Linux or `C:\Users\Username` on Windows.

**Comment by gurpindersingh3** [comment]
Error :
while tsh login --proxy=url --user=user@domain.com --auth =OneLogin
ERROR: 404 page not found

object not found

Solved after upgrading tsh client
